### PR TITLE
Fix app background color

### PR DESCRIPTION
**Pull request by imollov**
fixes:

* pages with different background color from header 

| before | after |
| ------ | ----- |
|Screenshot 2021-04-15 at 15 21 23 | Screenshot 2021-04-15 at 15 21 35|

* overscroll (solution suggested here https://github.com/aeternity/superhero-wallet/pull/931#discussion_r613115772)

| before | after |
| ------ | ----- |
|Screenshot 2021-04-15 at 13 12 08| Screenshot 2021-04-15 at 13 14 20 |

### CODE REVIEW

File: src/popup/App.vue [hunk 1]
```diff
@@ -117,6 +117,10 @@ body {
   background-color: $color-black;
 }
 
+body.bg {
```
Comment (davidyuk) [inline src/popup/App.vue:120]: the class name probably is too unspecific

**Review by davidyuk — APPROVED** [review]
lgtm

File: src/popup/App.vue (hunk 1 above)
Comment (imollov) [reply src/popup/App.vue:120]: Agreed. Suggestions?

Comment (imollov) [reply src/popup/App.vue:120]: how about `body.header-bg`?

Comment (imollov) [reply src/popup/App.vue:120]: or `body.main-bg`?

Comment (imollov) [reply src/popup/App.vue:120]: or `body.color-bg-3`?

**Pull request merged**